src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -1818,13 +1819,47 @@ export default abstract class ComparisonStore
         },
     });
 
+    readonly survivalSequencedMonths = remoteData<
+        { [uniquePatientKey: string]: number } | undefined
+    >({
+        await: () => [this.studies],
+        invoke: async () => {
+            const studyIds = this.studies.result!.map(s => s.studyId);
+            if (studyIds.length === 1 && studyIds[0] === 'nsclc_genie_bpc') {
+                const allTimelineData = _.flatten(
+                    await Promise.all(
```

Comment:
i would not nest await like this.     first assign the await to something and then flatten that.  it's too hard to see the asynchrouns thing